cli/src/factory_ops/unboxing/init

```diff
@@ -397,6 +394,8 @@ do
 done
 echo -e "Ok.\n"
 
+echo "Ensuring the cluster is in healthy state" 2>&1 | tee -a ${LOG_FILE}
+ensure_healthy_cluster
```

Comment:
where do we run `pcs resource cleanup`?

Reply:
okay we already do that before, found it.

Reply:
it's there at line# 366

Reply:
If not already logged, kindly log output of pcs status before and after the call to ensure_healthy_cluster. Thank you.

Reply:
done.